$OpenBSD: patch-plparse_totem-pl-parser_c,v 1.2 2009/12/14 11:49:51 ajacoutot Exp $

Use g_strstr_len() instead of GNU extension memmem.
git fc603d2f98802ae20f4902f2b1dd589b891595b6

--- plparse/totem-pl-parser.c.orig	Fri Dec  4 16:10:52 2009
+++ plparse/totem-pl-parser.c	Mon Dec 14 12:39:00 2009
@@ -1667,8 +1667,7 @@ totem_pl_parser_cleanup_xml (char *contents)
 		/* Broken file? */
 		if (end == NULL)
 			return;
-		if (memmem (needle, end - needle,
-			    "]]>", strlen ("]]>")) != NULL) {
+		if (g_strstr_len (needle, end - needle, "]]>") != NULL) {
 			/* Advance 3 and skip */
 			needle += 3;
 			continue;
